### DISCUSSION TITLE
HA OS Version

### DISCUSSION THREAD

**Discussion by pvillanyi** [blakeblackshear/frigate#3193] [Ask A Question]
Hi,
I would like to ask if I can safely upgrade HA OS?
A few months ago as I've started with frigate I had to downgrade to HA OS 6.6 to get Frigate working.
Is this problem solved?

**Comment by Ysbrand** [comment]
I'm running on Home Assistant OS 7.6 (x64) without any issue

**Reply by pvillanyi** [reply]
I've upgraded...
The cameras are working in general, thanks for the answer.
But person detection isn't working on all cameras...
